ISISNeutronMuon/MDANSE
# Fixed and added infrared jobs

**Description of work**
- Added infrared job which calculates the autocorrelation of the time derivative of the molecular dipole which is proportional to the IR intensity. See Eq. 10 from https://doi.org/10.1039/C3CP44302G.
- Dipoles in the dipole autocorrelation and infrared jobs are now calculated for the selected molecule and results are averaged over all instances of that molecule. 
- Dipoles are calculated for the molecule relative to its centre of mass.
- Added the charge mapping helper which is similar to the atom transmutation helper.

**Fixes**
Fixed the dipole autocorrelation jobs.
Fixed angularcorrelation job with a non default frame setting.

**To test**
- Check that the dipole autocorrelation function and Infrared job run successfully and check results plot successfully.
- Generate a trajectory of bulk H2O and run the IR calculation check that the IR results appear reasonable compared to the literature.
